<!DOCTYPE html>
<html>
    <head>
        <title>Wet Lab | engineering | iGem Winter Project</title>
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <link rel="icon" type="image/png" href="../../static/img/favicon.ico">
        <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
        <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
        <link rel="stylesheet" href="../../static/css/basic.css">
        <link rel="stylesheet" href="../../static/css/load.css">
        <link rel="stylesheet" href="../../static/css/pages.css">
        <link rel="stylesheet" href="../../static/css/components.css">
    </head>
    <body>
        <!-- scroll progress bar -->
        <div class="scroll-progress"></div>
        
        <!-- Loading cast -->
        <div class="loading-container">
            <div class="loading-spinner-image">
                <img src="../../static/img/loading.svg" alt="Loading...">
            </div>
            <div class="loading-text">Loading...</div>
        </div>

        <div class="top_bar">
            <div class="menu_btn">
                <ion-icon name="menu-outline"></ion-icon>
            </div>
            <div class="logo_wrap">
                <a href="../../index.html"><img id="team_logo" src="../../static/img/logo-apart-text.svg" alt=""></a>
            </div>
            <div class="menu_wrap">
                <div id="project" class="menu_item">
                    <div class="title">Project<ion-icon name="caret-back-outline"></ion-icon></div>
                    <ul class="choice">
                        <li><a href="../description.html">Description</a></li>
                        <li><a href="../design.html">Design</a></li>
                        <li><a href="../safety.html">Safety</a></li>
                    </ul>
                </div>
                <div id="wetlab" class="menu_item">
                    <div class="title">Wet Lab<ion-icon name="caret-back-outline"></ion-icon></div>
                    <ul class="choice">
                        <li><a href="./engineering.html">Engineering</a></li>
                        <li><a href="./construction.html">Construction</a></li>
                        <li><a href="./protocol.html">Protocol</a></li>
                        <li><a href="./parts.html">Parts</a></li>
                    </ul>
                </div>
                <div id="drylab" class="menu_item">
                    <div class="title">Dry Lab<ion-icon name="caret-back-outline"></ion-icon></div>
                    <ul class="choice">
                        <li><a href="../drylab/model&software.html">Model & Software</a></li>
                        <li><a href="../drylab/hardware.html">Hardware</a></li>
                    </ul>
                </div>
                <div id="human" class="menu_item">
                    <div class="title">Human Practice<ion-icon name="caret-back-outline"></ion-icon></div>
                    <ul class="choice">
                        <li><a href="../hp/overview.html">Overview</a></li>
                        <li><a href="../hp/education.html">Education</a></li>
                        <li><a href="../hp/entrepreneurship.html">Entrepreneurship</a></li>
                        <li><a href="../hp/integrated.html">Integrated Human Practices</a></li>
                        <li><a href="../hp/inclusivity.html">Inclusivity</a></li>
                        <li><a href="../hp/sustain.html">Sustainable Development</a></li>
                        <li><a href="../hp/ethical.html">Ethical Framework</a></li>
                    </ul>
                </div>
                <div id="team" class="menu_item">
                    <div class="title">Team<ion-icon name="caret-back-outline"></ion-icon></div>
                    <ul class="choice">
                        <li><a href="../team/attributions.html">Attributions</a></li>
                        <li><a href="../team/member.html">Member</a></li>
                    </ul>
                </div>
            </div>
        </div>
        
        <!-- Page Title Background -->
        <div class="page-title-bg">
            <div class="bg-image" style="background-image: url('../../static/img/engineer/peakpx.jpg')"></div>
            <div class="overlay"></div>
            <div class="title-content">
                <h1>Engineering</h1>
            </div>
        </div>

        <!-- Main content wrapper -->
        <div class="main">
            <!-- Sidebar wrapper -->
            <div class="sidebarBg">
                <div class="sidebar">
                    <!-- sidebar collapse button -->
                    <div class="sidebar-collapse-btn">
                        <ion-icon name="chevron-back-outline"></ion-icon>
                    </div>
                    
                    <div class="t1">
                        <span>Function Verification Module</span>
                        <div class="t2">
                            <a href="#cycle1">Cycle 1: Functional Verification of Each Tag</a>
                        </div>
                        <div class="t2">
                            <a href="#cycle2">Cycle 2: Functional Verification of the Fusion Protein</a>
                        </div>
                        <div class="t2">
                            <a href="#references1">References</a>
                        </div>
                    </div>

                    <div class="t1">
                        <span>Production Expression Verification Module</span>
                        <div class="t2">
                            <a href="#2cycle1">Cycle 1: Evaluation of plasmid expression</a>
                        </div>
                        <div class="t2">
                            <a href="#2cycle2">Cycle 2: Evaluating the expression of engineered bacteria in hydrogels</a>
                        </div>
                        <div class="t2">
                            <a href="#2cycle3">Cycle 3: Evaluating drug molecule permeability in an eye model</a>
                        </div>
                    </div>

                    <div class="t1">
                        <span>Safety Alert Module</span>
                        <div class="t2">
                            <a href="#3cycle1">Cycle 1: Validation of Co-expression of Crimson and RNF114</a>
                        </div>
                        <div class="t2">
                            <a href="#3cycle2">Cycle 2: Adjusting RBS to Regulate Expression Levels</a>
                        </div>
                        <div class="t2">
                            <a href="#3cycle3">Cycle 3: Correlation between Fluorescence and Safety Grades</a>
                        </div>
                        <div class="t2">
                            <a href="#references3">References</a>
                        </div>
                    </div>

                    <div class="t1">
                        <span>Hydrogel Module</span>
                        <div class="t2">
                            <a href="#4cycle1">Cycle 1: PVA-VS/PVA Accommodates Biofactory Testing</a>
                        </div>
                        <div class="t2">
                            <a href="#4cycle2">Cycle 2: PVA-VS/PVA Surface Modification</a>
                        </div>
                        <div class="t2">
                            <a href="#4cycle3">Cycle 3: pHEMA/Î²-CD-crHA Functional Test</a>
                        </div>
                        <div class="t2">
                            <a href="#references4">References</a>
                        </div>
                    </div>

                    <div class="t1">
                        <span>Killing Switch</span>
                        <div class="t2">
                            <a href="#5cycle1">Cycle 1: Phosphate Load and Release Testing</a>
                        </div>
                        <div class="t2">
                            <a href="#5cycle2">Cycle 2: Co-culture of engineered bacteria with phosphate to test suicide switch effectiveness</a>
                        </div>
                        <div class="t2">
                            <a href="#5references">References</a>
                        </div>
                    </div>

                    <div class="t1">
                        <span>Auxotrophic Mutual Assistance System</span>
                        <div class="t2">
                            <a href="#6cycle1">cycle1:Survival of strain A and strain B in the absence of relevant amino acids.</a>
                        </div>
                        <div class="t2">
                            <a href="#6cycle2">cycle2:Co-cultivation of strain A and strain B (at different dilution concentrations)</a>
                        </div>
                    </div>


                </div>
            </div>

            <!-- sidebar expand button -->
            <div class="sidebar-expand-btn">
                <ion-icon name="chevron-forward-outline"></ion-icon>
            </div>

            <!-- Content area -->
            <div class="content">
                <!-- Function Verification Module -->
                <section id="function-verification" class="parent-section">
                    <div class="section-header">
                        <h1>Function Verification Module</h1>
                        <button class="section-collapse-btn">
                            <ion-icon name="chevron-up-outline"></ion-icon>
                        </button>
                    </div>
                    <div class="section-content">
                        <div class="introduction">
                            <p>In this module, we systematically verify the functionality of our designed components through two main cycles of experiments and analyses.</p>
                        </div>

                        <section id="cycle1" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 1: Functional Verification of Each Tag</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>Design1</h3>
                                <p>We have found three functional peptides, Tat peptide, Gala3, and pelB, which promote cell permeability, endosomal escape, and exocytosis, and obtained a target peptide that specifically targets the Aquaporin 0 on lens cell through modeling software and AI-assisted design. In this cycle, we need to first fuse each tag with RNF114 through a linker to verify the effectiveness of each tag.</p>
                                <h3>Construction1</h3>
                                <p>In order to characterize the role of each part, we decided to construct the following gene pathways by chemical synthesis:</p>

                                <figure>
                                    <img src="../../static/img/engineer/synthesis.png" alt="Construction1">
                                    <!-- <figcaption>
                                        Fig.1 | Construction1
                                    </figcaption> -->
                                </figure>
                            
                                <h3>Test1</h3>
                                <p><strong>0. Verification the success of plasmid introduction</strong></p>
                                <p>Sanger sequencing; extraction of plasmid DNA (alkaline SDS separation); determination of DNA size and concentration by gel electrophoresis and Nanodrop.</p>

                                <p><strong>1. pelB tag</strong><sup><a href="#ref1">[1]</a></sup></p>
                                <ul>
                                    <li>A. Construction of plasmids, group 1: pelB-linker-RNF114-GPF; group 2: linker-RNF114-GPF; group 3: vector linearization and reconnection to obtain an empty vector, and then transformed into the target strain.</li>
                                    <li>B. Use a mild induction regimen (0.05 mM IPTG, 30â, 2hrs)to express and secrete proteins. Centrifuge and take the supernatant.</li>
                                    <li>C. Determine the relative molecular weight of the protein using 12% SDS-PAGE and perform quantitative analysis using ImageJ.</li>
                                </ul>

                                <p><strong>2. Tat peptide tag</strong><sup><a href="#ref2">[2]</a></sup></p>
                                <ul>
                                    <li>A. Construction of plasmids, group1: TAT-linker-RNF114-GPF; group2: linker-RNF114-GPF; group3: vector linearization and reconnection to obtain an empty vector, and then transformed into the target strain.</li>
                                    <li>B. Cultivate strains, extract and purify secretory proteins.</li>
                                    <li>C. Culture lens epithelial cells and co-incubate with the target protein for 1 h. Centrifuge at 1 000 x g for 10 min, collect cells from each well, wash three times with PBS, collect cells using FASCaliber flow cytometer, and analyze cell penetration using CellQuest software.</li>
                                </ul>

                                <p><strong>3. GALA3 tag</strong><sup><a href="#ref3">[3]</a></sup></p>
                                <ul>
                                    <li>A. Construction of plasmids, group1: GALA-linker-RNF114-GPF; group2: linker-RNF114-GPF; group3: vector linearization and reconnection to obtain an empty vector, and then transformed into the target strain.</li>
                                    <li>B. Cultivate strains, extract and purify secretory proteins.</li>
                                    <li>C. Lens epithelial cells were incubated with different concentrations (0.125, 0.25, and 1 Î¼M) of the protein for 12 h, co-stained with DAPI and observed by fluorescence microscopy, and the fluorescence intensity was quantified by flow cytometry.</li>
                                </ul>

                                <!-- <h5>Flow cytometry(FC)</h5>
                                <p>Cells were seeded in 6-well plates (Corning, NY, USA) at a density of 1 Ã 10<sup>6</sup> cells per well and cultured overnight. The culture medium was replaced with fresh culture medium containing recombinant proteins at the indicated concentrations. Following incubation, the cells were treated with trypsin, washed twice with PBS, collected, and finally resuspended in PBS. The fluorescence intensity was assessed using flow cytometry (Becton Dickinson, Franklin Lakes, NJ, USA). Data was analyzed using FlowJo 7.6 software (TreeStar, Ashland, Oregon, USA).</p> -->

                                <!-- <h5>Fluorescence microscopy observation</h5>
                                <p>Cells (1.0 Ã 10<sup style="color: var(--black);">4</sup> cells per well) were seeded in 24-well plates (Corning, NY, USA) and incubated at 37 Â°C overnight until 30% confluency. The cells were treated with recombinant proteins at the indicated concentrations. After incubation for 12 h, the cells were washed three times with precooled PBS (2.7 mM KCl, 137 mM NaCl, 10 mM Na2HPO4, and 2 mM KH2PO4), fixed with paraformaldehyde (PFA, 4%) (Sangon, Shanghai, China), and stained with 4,6-diamidino-2-phenylindole (DAPI) (Beyotime, Haimen, China) in the dark. The intracellular fluorescence distribution of recombinant proteins was imaged by confocal laser scanning microscopy (CLSM) (Nikon, Tokyo, Japan) and fluorescence microscopy (Nikon, Tokyo, Japan), and the data were analyzed using NLS software.</p> -->

                                <p><strong>4. Target peptide</strong><sup><a href="#ref4">[4]</a></sup></p>
                                <ul>
                                    <li>A. Construction of plasmids, group1: Targetpeptide-linker-RNF114-GPF; group2: Targetpeptide-linker-RNF114; group3: vector linearization and reconnection to obtain an empty vector, and then transformed into the target strain</li>
                                    <li>B. Cultivate strains, extract and purify secretory proteins</li>
                                    <li>C. Fluorescence polarization immunoassay to detect affinity: Add a certain amount of GFP-labeled fusion protein and AQP0 to the reaction system. Add different concentrations of unlabeled fusion protein to compete with GFP-labeled fusion protein for binding to AQP0, and fit the competition binding curve.</li>
                                </ul>

                                <h4>Learn1</h4>
                                <p>If all tags can play their corresponding functions, it means that the general idea of protein fusion is fine and the next step of overall fusion can be carried out. If the results do not meet expectations, redesign or search for alternative solutions and feedback, optimize the experimental methods before re-verify.</p>
                            </div>
                        </section>

                        <section id="cycle2" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 2: Functional Verification of the Fusion Protein</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>Design2</h3>
                                <p>Since our ultimate goal is to obtain a fusion protein with the functions of all the above tags, we decided to construct a gene pathway containing all peptide modifications in the second cycle and verify the functions of each part again. In order to optimize its spatial structure without affecting the domain (Ring-Finger structure) of the original protein, we continuously optimized the modification position and the length of the connecting peptide, and finally obtained the best and most qualified predicted protein structure and corresponding gene pathway.</p>

                                <h3>Construction2</h3>
                                <p>Through the elements that have been successfully constructed and verified in cycle 1, we can construct the gene of the complete fusion protein by PCR. The gene and hyaluronic acid synthesis gene pathway are as follows:</p>
                                <figure>
                                    <img src="../../static/img/engineer/cons2.png" alt="Fusion Protein Construction">
                                </figure>

                                <h3>Test2</h3>

                                <p><strong>0. Verification of successful plasmid introduction</strong>: DNA sanger sequencing, size and concentration detection</p>
                                
                                <p><strong>1. Expression of fusion protein</strong></p>

                                <p><strong>2. Extraction and purification of fusion protein</strong></p>

                                <p><strong>3. Verify the stability of fusion protein:</strong></p>
                                <p>Stability improvement plan: [Preferred plan (1), if the effect does not meet expectations, continue to further verify the plan (2)]</p>
                                <ol>
                                    <li>Hyaluronic acid encapsulation</li>
                                    <li>Rare amino acid replacement</li>
                                </ol>

                                <p><strong>Verification method:</strong></p>
                                <p>Hyaluronic acid and fusion protein (containing GFP tag) are added to the culture medium containing hydrolase in proportion (the specific concentration needs to be determined in advance), and bleached by a short light pulse. Bleaching causes the labeled protein to be divided into two subgroups - fluorescent protein and non-fluorescent protein. The fluorescence recovery rate after bleaching is related to the degradation rate. The fluorescence intensity is quantified by flow cytometry at intervals.<sup><a href="#ref5">[5]</a></sup></p>

                                <p><strong>Type and content of enzyme added:</strong><sup><a href="#ref6">[6]</sup></a><sup><a href="#ref7">[7]</a></sup></p>
                                <ul>
                                    <li>MMP1, 2, 9, 13</li>
                                    <li>Cathepsin A (in aqueous humor); Cathepsin V, K, B, S. Compared with Cathepsin S, Cathepsin V has the highest expression (about 200 times), followed by Cathepsin L and Cathepsin B (about 10 times and 5 times, respectively)</li>
                                </ul>

                                <p><strong>4. Verify the therapeutic function of the fusion protein</strong><sup><a href="#ref8">[8]</a></sup></p>
                                <ul>
                                    <li>A. Rat lens epithelial cells were placed in culture medium and incubated at 4Â°C for 24 hours to induce cataracts. 100 Î¼M fusion protein was added to the culture medium in the last 30 minutes of low temperature treatment in the experimental group, but not in the control group. The lenses were transferred to a 37Â°C environment for 30 minutes.</li>
                                    <li>B. After rewarming, the rat lens of the control group was placed in a dish containing extracellular solution and placed above the bottom light source of a Nikon SMZ18 microscope, and image A was taken. Subsequently, the light source was turned off and image B was taken to correct for ambient light interference. Then, the rat lens of the experimental group was placed above the bottom light source and image C was taken.</li>
                                    <li>C. Analyze the grayscale value of the lens area by ImageJ (NIH) and calculate the transmittance according to the following formula:</li>
                                </ul>
                                
                                <div class="math-block">
                                    Lens transmittance = \(\frac{C - B}{A - B} \times 100\%\)
                                </div>
                                

                                <h3>Learn2</h3>
                                <p>If the fusion protein is stable and its therapeutic function can be performed normally, it can be combined with hydrogel and hardware structure for the next cycle.</p>
                            </div>
                        </section>

                        <!-- References section -->
                        <section id="references1" class="sub-section">
                            <div class="section-header">
                                <h2>References</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <div class="reference-list">
                                    <div class="reference-item" id="ref1">
                                        <span class="ref-number">[1]</span>
                                        <span class="ref-content">Mirzadeh, K., Shilling, P.J., Elfageih, R., Cumming, A.J., Cui, H.L., Rennig, M., NÃ¸rholm, M.H.H., Daley, D.O. (2020). Increased production of periplasmic proteins in Escherichia coli by directed evolution of the translation initiation region. <em>Microb Cell Fact</em>, 19(1):85.</span>
                                        <a href="https://doi.org/10.1186/s12934-020-01339-8" class="ref-link" target="_blank">DOI: 10.1186/s12934-020-01339-8</a>
                                    </div>

                                    <div class="reference-item" id="ref2">
                                        <span class="ref-number">[2]</span>
                                        <span class="ref-content">è®¸éï¼èºæï¼ç°é¨é¦ï¼æå©å©ï¼å­£å®èï¼åéï¼å¤å£ï¼çèåï¼è®¸åæºªï¼éµå¯ç¥¥. (2008). ä¸¤ç§æ¹å¼å¶å¤çåæ ¸è¡¨è¾¾å«TATèç½è½¬å¯¼åèåèç½ç©¿èè½åæ¯è¾. <em>æ±èå¤§å­¦å­¦æ¥ï¼å»å­¦çï¼</em>, 18(6):466.</span>
                                    </div>

                                    <div class="reference-item" id="ref3">
                                        <span class="ref-number">[3]</span>
                                        <span class="ref-content">Li, C., Cao, X.W., Zhao, J., et al. (2020). Effective Therapeutic Drug Delivery by GALA3, an Endosomal Escape Peptide with Reduced Hydrophobicity. <em>J Membrane Biol</em>, 253:139-152.</span>
                                        <a href="https://doi.org/10.1007/s00232-020-00109-2" class="ref-link" target="_blank">DOI: 10.1007/s00232-020-00109-2</a>
                                    </div>

                                    <div class="reference-item" id="ref4">
                                        <span class="ref-number">[4]</span>
                                        <span class="ref-content">çå¼ºï¼çæ­å³°ï¼å¼ è±ä¾ ï¼ç­. (2024). è§ååæ¯åç«æ³æµå®æ°´äº§åä¸­çä¸é¦é. <em>åæè¯éªå®¤</em>, 43(03):307-313.</span>
                                        <a href="https://doi.org/10.13595/j.cnki.issn1000-0720.2022.111602" class="ref-link" target="_blank">DOI: 10.13595/j.cnki.issn1000-0720.2022.111602</a>
                                    </div>

                                    <div class="reference-item" id="ref5">
                                        <span class="ref-number">[5]</span>
                                        <span class="ref-content">Geva-Zatorsky, N., Issaeva, I., Mayo, A., Cohen, A., Dekel, E., Danon, T., Cohen, L., Liron, Y., Alon, U., & Eden, E. (2012). Using bleach-chase to measure protein half-lives in living cells. <em>Nature protocols</em>, 7(4):801-811.</span>
                                        <a href="https://doi.org/10.1038/nprot.2012.028" class="ref-link" target="_blank">DOI: 10.1038/nprot.2012.028</a>
                                    </div>

                                    <div class="reference-item" id="ref6">
                                        <span class="ref-number">[6]</span>
                                        <span class="ref-content">Sivak, J.M., & Fini, M.E. (2002). MMPs in the eye: emerging roles for matrix metalloproteinases in ocular physiology. <em>Prog Retin Eye Res</em>, 21(1):1-14.</span>
                                        <a href="https://doi.org/10.1016/s1350-9462(01)00015-5" class="ref-link" target="_blank">DOI: 10.1016/s1350-9462(01)00015-5</a>
                                    </div>

                                    <div class="reference-item" id="ref7">
                                        <span class="ref-number">[7]</span>
                                        <span class="ref-content">Coppini, L.P., Visniauskas, B., Costa, E.F., Filho, M.N., Rodrigues, E.B., Chagas, J.R., Farah, M.E., Barros, N.M., Carmona, A.K. (2015). Corneal angiogenesis modulation by cysteine cathepsins: In vitro and in vivo studies. <em>Exp Eye Res</em>, 134:39-46.</span>
                                        <a href="https://doi.org/10.1016/j.exer.2015.03.012" class="ref-link" target="_blank">DOI: 10.1016/j.exer.2015.03.012</a>
                                    </div>

                                    <div class="reference-item" id="ref8">
                                        <span class="ref-number">[8]</span>
                                        <span class="ref-content">Yang, H., Ping, X., Zhou, J., Ailifeire, H., Wu, J., Nadal-NicolÃ¡s, F.M., Miyagishima, K.J., Bao, J., Huang, Y., Cui, Y., Xing, X., Wang, S., Yao, K., Li, W., Shentu, X. (2024). Reversible cold-induced lens opacity in a hibernator reveals a molecular target for treating cataracts. <em>J Clin Invest</em>, 134(18):e169666.</span>
                                        <a href="https://doi.org/10.1172/JCI169666" class="ref-link" target="_blank">DOI: 10.1172/JCI169666</a>
                                    </div>
                                </div>
                            </div>
                        </section>
                    </div>
                </section>
                <!-- Function Verification Module End -->

                <!-- Production Expression Verification Module -->
                <section id="production-expression" class="parent-section">
                    <div class="section-header">
                        <h1>Production Expression Verification Module</h1>
                        <button class="section-collapse-btn">
                            <ion-icon name="chevron-up-outline"></ion-icon>
                        </button>
                    </div>
                    <div class="section-content">
                        <!-- <div class="introduction">
                            <p>This module focuses on verifying the expression efficiency and stability of our engineered bacteria system in different conditions and environments.</p>
                        </div> -->

                        <section id="2cycle1" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 1: Evaluation of plasmid expression</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>The expression of eukaryotic proteins in prokaryotic systems typically requires lower culture temperatures to reduce inclusion body formation and promote proper protein folding. The average surface temperature of the eye is 34.14Â°C Â± 1.51Â°C at room temperature and varies with ambient temperature. Since we aim to enable the engineered bacteria to continuously express the RNF114 protein in the ocular environment, it is necessary to experimentally evaluate the expression efficiency of the recombinant plasmid in engineered bacteria under different conditions. To enhance the stability of RNF114, we have designed a scheme for the synchronous release of polyols; thus, concurrent experiments are required to adjust the expression levels of HA in the engineered bacteria and the ratio of the two proteins by testing different strengths of RBS (ribosome binding sites).</p>
                                <p>The plasmids expressing recombinant RNF114 and HA were simultaneously introduced into the engineered bacteria. Cultures were set up at three different temperatures: 34Â°C, 30Â°C, and 26Â°C. The contents of HA and recombinant RNF114 in the culture medium were detected at regular intervals, and production curves were plotted.</p>

                                <h3>2. Construction</h3>
                                <p>The recombinant plasmid pET-32a(+)-RNF114-crimson was constructed, with both the recombinant RNF114 and the fluorescent protein Crimson under the control of the T7 promoter. The plasmid pET-32a(+)-hasAB was also constructed. These plasmids were introduced into competent Shuffle T7-B Escherichia coli cells via Bacterial Transformation and then streaked on agar plates for recovery. The transformed cells were cultured in media containing ampicillin to select for positive colonies, which were then picked for further cultivation.</p>
                                <!-- <figure>
                                    <img src="../../static/img/engineer/construction-grid.png" alt="Construction Image 1">
                                </figure> -->
                                <div class="img-grid">
                                    <img src="../../static/img/engineer/cons-left.png" alt="Construction Image 1">
                                    <img src="../../static/img/engineer/cons-right.png" alt="Construction Image 2">
                                </div>

                                <h3>3. Test</h3>
                                <ol>
                                    <li><p><strong>Bacteria Cultivation:</strong> Positive colonies were picked for small-scale shake-flask cultivation in BHI (Brain Heart Infusion) broth, with an equal inoculum in each tube.  Incubate at 34Â°C, 30Â°C and 26Â°C for 14 days and change the medium on the 7th day. Supernatants were collected at different time points between 3 h and 14 days. Separate RNF-114 from HA using His-tag affinity chromatography.</p></li>
                                    <li><p><strong>Detection of HA Content:</strong> After separation of HA according to the protocol, the content of HA in the range of 30-70 kDa was detected using Gel Permeation Chromatography (GPC), and a GPC curve was plotted.</p></li>
                                    <li><p><strong>Detection of RNF-114 Content:</strong> The concentration of RNF-114 was determined by measuring the absorbance at 280 nm using UV spectrophotometry, and a curve representing the content of secreted protein was plotted.</p></li>
                                </ol>

                                <h3>4. Learn</h3>
                                <p>To address the potential issue of insufficient RNF114 expression, we aim to introduce molecular chaperone proteins as a solution. After searching protein interaction databases, we identified that RNF114 interacts with heat shock protein HSP90AA1 and HSP70 member 5. One of the key functions of the HSP family is to act as molecular chaperones, facilitating the proper folding of proteins. Therefore, we hypothesize that HSP90 and HSP70 could serve as molecular chaperones for RNF114.</p>
                                <p>Due to the lack of experimental data on RNF114 expression in prokaryotic systems, the expression efficiency of RNF114 in E. coli at around 34Â°C is unknown. Moreover, to maximize the expression efficiency of the target protein RNF114, we prefer to minimize the expression of exogenous proteins in the engineered bacteria. Thus, we decided to prioritize genetic modifications without co-expressing chaperone proteins in our experiments. If RNF114 is expressed correctly and in sufficient quantities without chaperones, we will not introduce additional plasmids for co-expression. However, if the expression is suboptimal, we will introduce plasmids to co-express the molecular chaperones HSP90 or HSP70.</p>

                            </div>
                        </section>

                        <section id="2cycle2" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 2: Evaluating the expression of engineered bacteria in hydrogels</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We plan to use hydrogel as a vector for engineered bacteria, so after confirming the effect of plasmid expression, we need to confirm the expression of engineered bacteria in hydrogel.</p>

                                <h3>2. Construction</h3>
                                <ol>
                                    <li><p><strong>Option1:</strong> A square frame 5% w/v PVA-VS/PVA 95:5 hydrogel containing engineered bacteria was moulded, with a side length of 8mm, and a thickness of 1mm. Next, the square frame was embedded in a square 10% w/v PVA-VS hydrogel. The hydrogel's sides measured 12mm in length and 1.5mm in thickness.</p></li>
                                    <li><p><strong>Option2:</strong> A pHEMA/Î²-CD-crHA hydrogel with MA substitution of 20% and m(20)HA-Î²-CD content of 8% was constructed, and contact lenses containing biofactories were sequentially fabricated.</p></li>
                                </ol>

                                <h3>3. Test</h3>
                                <ol>
                                    <li><p><strong>Bacteria Cultivation:</strong> After picking the positive bacteria for small-scale shake-flask culture, incubate them in BHI medium and inoculate each tube with the same amount of bacteria. Incubate at 34Â°C, 30Â°C and 26Â°C for 14 days and change the medium on the 7th day. Supernatants were collected at different time points between 3 h and 14 days. Separate RNF-114 from HA using His-tag affinity chromatography.</p></li>
                                    <li><p><strong>Determination of HA content:</strong> After the separation of HA according to the protocol, the HA content of 30-70 kDa was detected by Gel Permeation Chromatography (GPC), and the GPC curve was plotted.</p></li>
                                    <li><p><strong>Detection of RNF-114 content:</strong> The absorbance at 280 nm was measured by UV spectrophotometry, the protein concentration was calculated, and the secretory protein content curve was plotted.</p></li>
                                </ol>

                                <h3>4. Learn</h3>
                                <p>The pore size and surface modification of hydrogels can, on the one hand, fulfill the function of slow drug release, but on the other hand, they may also affect the release of fusion proteins. For example, in the PVA-VA hydrogel, we need to adjust the area of surface modification to find the optimal point of fusion protein release.</p>
                            </div>
                        </section>

                        <section id="2cycle3" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 3: Evaluating drug molecule permeability in an eye model</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We wanted to validate the drug delivery efficiency of CLearCat in a real ocular environment, so we designed an in vitro eye simulation chip and embedded the engineered bacteria in a hydrogel, administered the drug in a contact lens mode, and tested the bioavailability of RNF-114 in the simulated eye.</p>

                                <h3>2. Construction</h3>
                                <p>The eyeball simulation chip is described in the <a href="../drylab/hardware.html#eyeball-simulation">Hardware - Eyeball Simulation</a> section. We covered the outer tear layer with contact lenses embedded with engineered bacteria to detect the amount of drug that eventually reaches the lens site.</p>

                                <h3>3. Test</h3>
                                <p>The constructed bacterial hydrogel was covered on a running eyeball simulator and continued to be administered for 14 days, detecting the activity of the engineered bacteria every 24 hours.Drugs were collected from the bottom of the simulator at different time points between 3 h and 14 days in the following manner: at the collection time point, the circulation of aqueous humor was suspended by a controlled fluid pump, and then the aqueous humor was aspirated from one side and assayed for RNF-114 drug content using Western Blot.</p>

                                <h3>4. Learn</h3>
                                <p>The data was processed to allow a simple comparison of the drug content of Cycle1, Cycle2 and Cycle3.The tear layer, corneal layer and aqueous humor circulation can substantially limit drug delivery. And we replaced animal experiments by constructing an in vitro eye simulation to test the amount of drug that can eventually reach the lens.</p>
                            </div>
                        </section>
                    </div>
                </section>
                <!-- Production Expression Verification Module End -->

                <!-- Safety Alert Module Start -->
                <section id="safety-alert" class="parent-section">
                    <div class="section-header">
                        <h1>Safety Alert Module</h1>
                        <button class="section-collapse-btn">
                            <ion-icon name="chevron-up-outline"></ion-icon>
                        </button>
                    </div>
                    <div class="section-content">
                        <section id="3cycle1" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 1: Validation of Co-expression of Crimson and RNF114</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We aim to utilize the naturally occurring polycistronicmRNA structure in prokaryotes to achieve synchronous expression of RNF114 and the fluorescent protein Crimson. Both coding genes share a single promoter. After transcription, the two coding genes are located on the same mRNA, thereby enabling synchronous expression of the two genes.</p>

                                <h3>2. Construction</h3>
                                <p>A recombinant plasmid pET-32a(+)-RNF114-GFP-Crimson was constructed, with both the recombinant RNF114 and the fluorescent protein Crimson placed under the control of the T7 promoter. The plasmid was introduced into competent Shuffle T7-B Escherichia coli cells via heat-shock transformation and then streaked for recovery. The transformed cells were cultured on a medium containing ampicillin (Ampicillin). Positive colonies were selected and transferred to liquid culture at 33Â°C.</p>
                                <figure>
                                    <img src="../../static/img/engineer/alert-cons.png" alt="Construction Image">
                                </figure>

                                <h3>3. Test</h3>
                                <p><strong>Sampling Schedule:</strong> Bacterial cultures were sampled every 24 hours for protein expression detection after the start of cultivation.For each sample, a fixed volume of bacterial culture was taken. The supernatant was collected to detect RNF114, while the pellet was collected to detect Crimson.</p>
                                <p><strong>Protein Expression Detection:</strong></p>
                                <ul>
                                    <li><p><strong>Fluorescent Protein Expression Detection:</strong> Fluorescence emission of green fluorescent protein (GFP) and red fluorescent protein Crimson was measured using a fluorescence spectrophotometer. The excitation wavelength for Crimson was 588 nm with an emission wavelength of 617 nm, while the excitation wavelength for GFP was 488 nm with an emission wavelength of 507 nm. The experimental results were analyzed based on the fluorescence intensity values (Fluorescence intensity = Fluorescence value / OD600).<sup><a href="#3ref1">[1]</a><a href="#3ref3">[3]</a></sup></p></li>
                                    <li><p><strong>RNF114 Expression Detection and Relative Expression Levels:</strong> Protein expression was analyzed by SDS-PAGE, followed by Coomassie Brilliant Blue staining of the gel. The expression levels of RNF114 and Crimson were compared to confirm synchronous expression of the two proteins. The molecular weight of Crimson is 27 kDa, while the modified RNF114 has a molecular weight of approximately 35 kDa.By measuring the OD600 value, the same amount of bacterial culture is ensured for each detection.</p></li>
                                </ul>

                                <h3>4. Learn</h3>
                                <p>Our experiments successfully validated the synchronous expression of Crimson and RNF114. We learned that in a polycistronic structure, the ribosome binding site (RBS) affects the reading of the open reading frame (ORF) through its complementarity to the ribosome and its distance from the start codon. Therefore, we plan to adjust the RBS sequences upstream of the two coding genes. By using RBS sequences of varying strengths, we aim to maximize the expression of RNF114 while ensuring that fluorescence signals meet the measurement requirements.</p>
                            </div>
                        </section>

                        <section id="3cycle2" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 2: Adjusting RBS to Regulate the Relative Expression Levels</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We designed ribosome binding sites (RBS) of varying strengths for the two coding genes to achieve higher expression levels of RNF114, thereby directing translational resources primarily towards the expression of the target therapeutic molecule. The RBS sequences were sourced from the <a href="https://parts.igem.org/Ribosome_Binding_Sites/Prokaryotic/Constitutive/Anderson" target="_blank">Anderson RBS family database</a>. This database provides 40 available RBS sequences for Escherichia coli and includes strength information for some of the RBS sequences. Based on experimental results, we will test different combinations of RBS to identify the optimal one that maximizes RNF114 expression while ensuring that the fluorescence signals meet the measurement requirements.</p>

                                <h3>2. Construction</h3>
                                <p>We constructed a plasmid based on the pET-32a(+)-RNF114-GFP-Crimson recombinant plasmid. The RBS upstream of the Crimson coding sequence was BBa_J61107, which has moderate expression strength, while the RBS upstream of the RNF114 coding sequence was RBS1, which has a stronger expression strength. The same plasmid transformation and positive colony cultivation procedures were followed. Additionally, engineered bacteria were encapsulated in hydrogels and cultured in a simulated tear fluid environment, with samples taken every 48 hours for protein detection.</p>

                                <h3>3. Test</h3>
                                <p><strong>Sampling Schedule:</strong> Bacterial cultures were sampled every 24 hours for protein expression detection after the start of cultivation.For each sample, a fixed volume of bacterial culture was taken. The supernatant was collected to detect RNF114, while the pellet was collected to detect Crimson.</p>
                                <ul>
                                    <li><p><strong>Fluorescent Protein Expression Detection:</strong> Fluorescence emission of green fluorescent protein (GFP) and red fluorescent protein Crimson was measured using a fluorescence spectrophotometer. The excitation wavelength for Crimson was 588 nm with an emission wavelength of 617 nm, while the excitation wavelength for GFP was 488 nm with an emission wavelength of 507 nm. Experimental results were analyzed based on fluorescence intensity values (Fluorescence intensity = Fluorescence value / OD600).</p></li>
                                    <li><p><strong>RNF114 Expression Detection and Relative Expression Levels:</strong>Protein expression was analyzed by SDS-PAGE, followed by Coomassie Brilliant Blue staining of the gel. The expression levels of RNF114 and Crimson were compared to determine their relative expression levels.By measuring the OD600 value, the same amount of bacterial culture is ensured for each detection.</p></li>
                                    <li><p><strong>Encapsulated Engineered Bacteria in Hydrogels:</strong> The primary focus was to verify whether the fluorescence signals from the engineered bacteria encapsulated in hydrogels could meet the detection requirements.<sup><a href="#3ref2">[2]</a></sup></p></li>
                                    <li><p><strong>Cell Viability Detection:</strong>A cell viability detection device was used to measure the fluorescence signals from the engineered bacteria encapsulated in hydrogels(Learn more at <a href="../drylab/hardware.html">Hardware - Cell Viability Detection</a>). Upon pressing the button, an Arduino drives the PT4115 to emit light of a specific wavelength from an LED. The light intensity is controlled by a rotary encoder, passes through a filter to reach the fluorescent protein, and is then detected by a BH1750 sensor, with the light intensity displayed on an LCD screen.</p></li>
                                </ul>

                                <h3>4. Learn</h3>
                                <p>Under the premise of ensuring sufficient RNF114 expression, the fluorescence detection device enabled real-time monitoring of the engineered bacteria's ability to produce RNF114. Considering that the viability of the engineered bacteria in practical applications may be affected by various factors, including storage conditions and duration before use, temperature and nutritional conditions in the ocular microenvironment during use, and frequency of use, the duration of therapeutic efficacy may vary among different users. Additionally, the death and lysis of engineered bacteria after viability loss may have adverse effects on the user's eyes. Therefore, we aim to establish a unified safety standard for users to guide them in timely replacing ineffective contact lenses.</p>
                            </div>
                        </section>

                        <section id="3cycle3" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 3: Determination of the Correlation between Fluorescence and Safety Grades</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We aim to generate a warning signal when the cell viability decreases to a certain level and the production of RNF114 is significantly reduced. Therefore, it is necessary to establish a quantifiable correlation between cell viability and fluorescence signal intensity. We monitored the live cell density of engineered bacteria and the corresponding fluorescence signal intensity in a simulated ocular environment. By analyzing the fluorescence data of engineered bacteria from the logarithmic phase, stationary phase, to the apoptotic phase, we determined the fluorescence intensity thresholds corresponding to three safety grades: "Safe," "Recommended for Replacement," and "Not Usable."</p>

                                <h3>2. Construction</h3>
                                <p>The pET-32a(+)-RNF114-Crimson recombinant plasmid and pET-32a(+)-hasAB recombinant plasmid, which were optimized in Cycle 2, were transformed into the engineered bacteria. The engineered bacteria were encapsulated in hydrogels and cultured in BHI broth, with the medium being replaced every 7 days.</p>

                                <h3>3. Test</h3>
                                <p>Every 24 hours, a fixed-size sample from the bacterial encapsulation area was taken for fluorescence intensity detection. Simultaneously, an equally sized sample was taken for live cell volume fraction detection.</p>
                                <ul>
                                    <li><p><strong>Fluorescence Intensity Detection:</strong>The fluorescence signal of Crimson was detected using the detection device in the contact lens care box.</p></li>
                                    <li><p><strong>Live Cell Volume Fraction Detection:</strong>ATP Quantification Assay (CellTiter-Glo 3D Cell Viability Assay). The ATP content of the encapsulated bacteria in the hydrogel was measured using the CellTiter-Glo 3D Cell Viability Assay Kit (Promega, USA). At each time point, the culture medium on top of the hydrogel was removed, and CellTiter-Glo 3D reagent was added to the samples according to the manufacturer's instructions. The samples were incubated with shaking in the dark at room temperature for 30 minutes, and then the fluorescence signal of the samples was measured using a multi-functional plate reader. The luminescence intensity was converted to ATP content through a standard curve, thereby quantitatively analyzing the ATP content in the samples. The ATP content of the samples reflects cell viability.</p></li>
                                </ul>

                                <h3>4. Learn</h3>
                                <p>By synchronously monitoring the Crimson fluorescence signal and cell viability, we established the relationship between fluorescence intensity and cell viability when a fixed amount of engineered bacteria was inoculated. The fluorescence intensity value when the cell viability of engineered bacteria was â¥80% of the maximum was set as the threshold for the "Safe" grade. The fluorescence intensity value when the cell viability was â¤80% but â¥50% of the maximum was set as the boundary value for the "Recommended for Replacement" grade. The fluorescence intensity value when the cell viability was â¤50% of the maximum was set as the threshold for the "Not Usable" grade.</p>
                            </div>
                        </section>

                        <section id="references3" class="sub-section">
                            <div class="section-header">
                                <h2>References</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <div class="reference-list">
                                    <div class="reference-item" id="3ref1">
                                        <span class="ref-number">[1]</span>
                                        <span class="ref-content">Ning L, Geng Y, Lovett-Barron M, et al. A Bright, Nontoxic, and Non-aggregating red Fluorescent Protein for Long-Term Labeling of Fine Structures in Neurons. <em>Front Cell Dev Biol</em>, 2022, 10: 893468.</span>
                                        <a href="https://doi.org/10.3389/fcell.2022.893468" class="ref-link" target="_blank">DOI: 10.3389/fcell.2022.893468</a>
                                    </div>

                                    <div class="reference-item" id="3ref2">
                                        <span class="ref-number">[2]</span>
                                        <span class="ref-content">Puertas-BartolomÃ© M, GutiÃ©rrez-Urrutia I, Teruel-Enrico L L, et al. Self-Lubricating, Living Contact Lenses. <em>Advanced Materials</em>, 2024, 36(27): 2313848.</span>
                                        <a href="https://doi.org/10.1002/adma.202313848" class="ref-link" target="_blank">DOI: 10.1002/adma.202313848</a>
                                    </div>

                                    <div class="reference-item" id="3ref3">
                                        <span class="ref-number">[3]</span>
                                        <span class="ref-content">å®ç«å¨. å¤§è æèåé¡ºåå­è¡¨è¾¾è½½ä½ä¸­ååºå è¡¨è¾¾æ°´å¹³çç ç©¶[D]. å¤©æ´¥:å¤©æ´¥å¤§å­¦, 2007.</span>
                                        <a href="https://d.wanfangdata.com.cn/thesis/Y1359970" class="ref-link" target="_blank">DOI: 10.7666/d.Y1359970</a>
                                    </div>
                                </div>
                            </div>
                        </section>
                    </div>
                </section>
                <!-- Safety Alert Module End -->

                <!-- Hydrogel Module -->
                <section id="hydrogel-module" class="parent-section">
                    <div class="section-header">
                        <h1>Hydrogel Module</h1>
                        <button class="section-collapse-btn">
                            <ion-icon name="chevron-up-outline"></ion-icon>
                        </button>
                    </div>
                    <div class="section-content">
                        <section id="4cycle1" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 1: PVA-VS/PVA Accommodates Biofactory Testing</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>The objective was to explore the most effective synthesis scheme for hydrogel-based contact lenses. Initially, we developed contact lens 1.0 in accordance with the existing literature to facilitate the operation of engineered bacteria in non-visual areas and ensure the functionality of biological factories.<sup><a href="#4ref1">[1]</a></sup></p>

                                <h3>2. Construction</h3>
                                <p>A square frame 5% w/v PVA-VS/PVA 95:5 hydrogel containing engineered bacteria was moulded, with a side length of 8mm, and a thickness of 1mm. Next, the square frame was embedded in a square 10% w/v PVA-VS hydrogel. The hydrogel's sides measured 12mm in length and 1.5mm in thickness.</p>

                                <h3>3. Test</h3>
                                <p>The hydrogel containing the engineered bacteria was placed into BHI medium and cultured for 24, 48, and 72 hours, respectively. The culture medium was enriched with simulated tears and lysozyme. Imaging was performed using a LIVE/DEAD BacLight bacterial viability assay kit (Thermo Fisher Scientific L7012) with a confocal microscope.</p>

                                <ul>
                                    <li><p><strong>Measuring the activity of the engineered bacteria:</strong> a significant decrease in bacterial activity is expected.</p></li>
                                    <li><p><strong>The restriction of the hydrogel to the engineered bacteria</strong> is also evident, with no engineered bacteria expected to grow outside the restricted area and clear boundaries visible in the hydrogel.</p></li>
                                    <li><p><strong>The release of fusion protein</strong> will be detected using the following methods: the medium will be collected in fractions and subjected to SDS-PAGE and scanning grey scale analysis. Expectation: The release of the fusion protein should be continuous and visible in the collected fraction.</p></li>
                                    <li><p><strong>Test for cytocompatibility:</strong> a gradual decrease in cell activity is expected.</p></li>
                                </ul>

                                <h3>4. Learn</h3>
                                <p>Although contact lens 1.0 can effectively limit the biofactory, it is difficult to block the exchange of lysozyme and bacterial metabolites. We have discovered that the physical and chemical properties of hydrogels can be modified by surface modification to achieve the desired result.<sup><a href="#4ref3">[3]</a></sup><sup><a href="#4ref4">[4]</a></sup></p>
                            </div>
                        </section>

                        <section id="4cycle2" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 2: PVA-VS/PVA Surface Modification</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>Based on contact lens 1.0, we made some surface modifications to the hydrogel to block the exchange of substances other than fusion proteins and constructed contact lens 2.0.</p>

                                <h3>2. Construction</h3>
                                <p>The PVA hydrogel was modified with surface polyethylene glycol (PEG) molecules and negatively charged acrylic acid (AA) monomers to create a hydrophilic surface and reduce protein adhesion.</p>

                                <h3>3. Test</h3>
                                <ul>
                                    <li><p><strong>Measurement of engineered bacterial activity:</strong> No significant decrease in bacterial activity is expected and a significant improvement over Cycle 1.</p></li>
                                    <li><p><strong>Measurement of fusion protein release ability:</strong> Collect supernatant in batches and perform SDS-PAGE followed by grey scale scanning analysis. Expectation: Sustained release of fusion protein should be detectable in the supernatant. There may be a slight decrease compared to Cycle 1.</p></li>
                                    <li><p><strong>Detection of cytocompatibility:</strong> A slight decrease in cellular activity and a significant improvement over Cycle 1 are expected.</p></li>
                                </ul>

                                <h3>4. Learn</h3>
                                <p>Subsequently, it is necessary to find a balance between hydrogel surface modification and fusion protein release and adjust the coverage area of the surface modification to reduce the impact on fusion protein release. Considering that CLearCat is a material used in the eye, we must ensure that the visual area is transparent and oxygen permeable to ensure the comfort of contact lens wear.</p>
                            </div>
                        </section>

                        <section id="4cycle3" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 3: pHEMA/Î²-CD-crHA Functional Test</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>Given the good antimicrobial and anti-protein adsorption properties of pHEMA/Î²-CD-crHA material, we constructed contact lenses using a single material.</p>

                                <h3>2. Construction</h3>
                                <p>A pHEMA/Î²-CD-crHA hydrogel with MA substitution of 20% and m(20)HA-Î²-CD content of 8% was constructed, and contact lenses containing biofactories were sequentially fabricated.<sup><a href="#4ref2">[2]</a></sup></p>

                                <h3>3. Test</h3>
                                <p>The hydrogels containing engineered bacteria were placed in BHI medium and incubated for 24, 48, and 72 h. Artificial simulated tears containing lysozyme were added to the medium. Bacterial viability was measured using the LIVE/DEAD BacLight Bacterial Viability Assay Kit (Thermo Fisher Scientific L7012) with confocal microscope imaging.</p>

                                <ul>
                                    <li><p><strong>Measurement of engineered bacterial activity:</strong> A light decrease in bacterial activity is expected.</p></li>
                                    <li><p><strong>Measurement of restriction of engineered bacteria by hydrogel:</strong> No growth of engineered bacteria is expected except in restricted areas with clear boundaries visible in the hydrogel.</p></li>
                                    <li><p><strong>Detect ability to release fusion protein:</strong> Collect supernatant in batches and perform SDS-PAGE followed by scanning grey scale analysis. Expectation: Continuous release of fusion protein can be detected in the supernatant.</p></li>
                                    <li><p><strong>Detection of cytocompatibility:</strong> A slight decrease in cell activity is expected.</p></li>
                                </ul>

                                <h3>4. Learn</h3>
                                <p>In this cycle, we have verified that pHEMA/Î²-CD-crHA can indeed be used as a manufacturing material for contact lenses but may affect the release of fusion proteins. Therefore, the assigned concentration of the constituent materials must be continuously adjusted.</p>
                            </div>
                        </section>

                        <!-- Add references to existing references section -->
                        <section id="references4" class="sub-section">
                            <div class="section-header">
                                <h2>References</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <div class="reference-list">
                                    <div class="reference-item" id="4ref1">
                                        <span class="ref-number">[1]</span>
                                        <span class="ref-content">Puertas-BartolomÃ© M, GutiÃ©rrez-Urrutia I, Teruel-Enrico L L, et al. Self-Lubricating, Living Contact Lenses.[J]. Advanced materials (Deerfield Beach, Fla.), 2024, 36(27): e2313848.</span>
                                        <a href="https://doi.org/10.1002/adma.202313848" class="ref-link" target="_blank">DOI: 10.1002/adma.202313848</a>
                                    </div>

                                    <div class="reference-item" id="4ref2">
                                        <span class="ref-number">[2]</span>
                                        <span class="ref-content">Deng H, Zhang X, Su S, et al. Poly(2-hydroxyethyl methacrylate-co-methacrylated hyaluronan-Î²-cyclodextrin) hydrogel: A potential contact lens material with high hydrophilicity, good mechanical properties and sustained drug delivery.[J]. International journal of biological macromolecules, 2024, 283(Pt 1): 137579.</span>
                                        <a href="https://doi.org/10.1016/j.ijbiomac.2024.137579" class="ref-link" target="_blank">DOI: 10.1016/j.ijbiomac.2024.137579</a>
                                    </div>

                                    <div class="reference-item" id="4ref3">
                                        <span class="ref-number">[3]</span>
                                        <span class="ref-content">Rao L, Liu Y, Zhou H. Significantly improved cell affinity of polydimethylsiloxane enabled by a surface-modified strategy with chemical coupling.[J]. Journal of materials science. Materials in medicine, 2022, 33(10): 66.</span>
                                        <a href="https://doi.org/10.1007/s10856-022-06690-3" class="ref-link" target="_blank">DOI: 10.1007/s10856-022-06690-3</a>
                                    </div>

                                    <div class="reference-item" id="4ref4">
                                        <span class="ref-number">[4]</span>
                                        <span class="ref-content">Drozdova M G, Zaytseva-Zotova D S, Akasov R A, et al. Macroporous modified poly (vinyl alcohol) hydrogels with charged groups for tissue engineering: Preparation and in vitro evaluation.[J]. Materials science & engineering. C, Materials for biological applications, 2017, 75: 1075-1082.</span>
                                        <a href="https://doi.org/10.1016/j.msec.2017.03.017" class="ref-link" target="_blank">DOI: 10.1016/j.msec.2017.03.017</a>
                                    </div>
                                </div>
                            </div>
                        </section>

                    </div>
                </section>

                <section id="KillingSwitch" class="parent-section">
                    <div class="section-header">
                        <h1>Killing Switch</h1>
                        <button class="section-collapse-btn">
                            <ion-icon name="chevron-up-outline"></ion-icon>
                        </button>
                    </div>
                    <div class="section-content">
                        <section id="5cycle1" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 1: Phosphate Load and Release Testing</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We first examined the phosphate loading of poly (2-hydroxyethyl methacrylate-methacrylate copolymer-hyaluronic acid-Î²-cyclodextrin) hydrogels and examined their phosphate limitation.</p>

                                <h3>2. Construction</h3>
                                <p>A pHEMA/Î²-CD-crHA hydrogel with MA substitution of 20% and m(20)HA-Î²-CD content of 11% was constructed. A certain mass (m) of dry gel (1 cm Ã 1 cm Ã 1 mm) was immersed in 2 mL (V<sub>0</sub>) of phosphate solution (1 mg/mL in deionized water, C<sub>0</sub>) and incubated in an incubator (100 rpm) at 37 Â°C for 48 hours protected from light.</p>
                                <p>After wiping off the free phosphate solution from the surface, the loaded hydrogel was placed in 6 mL of deionized water and incubated in an incubator (100 rpm) at 37 Â°C for 48 hours protected from light. At designed intervals, 3 mL of release medium was collected and refreshed with 3mL of deionized water.</p>

                                <h3>3. Test</h3>
                                <ol>
                                    <li>
                                        <p><strong>Load detection:</strong> After removing the loaded hydrogel, the volume and concentration of residual phosphate solution were recorded as V<sub>t</sub> and C<sub>t</sub>. The absorbance of residual phosphate solution was measured at 880 nm in a spectrophotometer using ammonium molybdate spectrophotometry. The loading capacity (DLC) was evaluated using the following equation:</p>
                                        <div class="math-block">
                                            $$DLC = \frac{V_0\times C_0-V_t\times C_t}{m}$$
                                        </div>
                                    </li>
                                    <li>
                                        <p><strong>Release detection:</strong> Measurement of the absorbance of the collected solution by spectrophotometry. The percentage of phosphate release is calculated from the ratio of the released phosphate to the total amount loaded in the hydrogel.</p>
                                    </li>
                                </ol>

                                <h3>4. Learn</h3>
                                <p>We would like to see a phosphate-loaded zone with an inner diameter of 9 mm, an outer diameter of 11 mm, and a thickness of 1 mm form in the contact lens-type hydrogel, with an initial concentration of 1.82 mmol/L. If the hydrogel can be maximally loaded beyond this concentration and still maintain a high concentration of phosphate over a period of time, it could support the design of a suicide switch. Similarly, we can change the design concentration of phosphate by adjusting the size of the phosphate loading zone.</p>
                            </div>
                        </section>

                        <section id="5cycle2" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 2: Co-culture of engineered bacteria with phosphate to test suicide switch effectiveness</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We have verified the effective loading of the hydrogel to phosphate, and subsequently, we wish to test the effectiveness of this suicide system by detecting the initiation threshold of the suicide switch PHO5 promoter.</p>

                                <h3>2. Construction</h3>
                                <p>The <strong>pET-32a(+)-p_PhoB-mazF recombinant plasmid</strong> was constructed and the suicide gene mazF was placed under the control of the phosphate-sensitive promoter PHO5. The plasmid was introduced by heat-excited transformation into receptorized SHuffle T7-B E. coli that had been introduced with pET-32a(+)-RNF114-crimso, and recovery was carried out by streak culture. The positive bacteria were cultured in medium containing ampicillin (Ampicillin), picked and transferred to liquid medium at 33Â°C.</p>
                                
                                <div class="img-grid-with-caption">
                                    <div class="img-grid">
                                        <img src="../../static/img/engineer/7577left.png" alt="Construction Image 1">
                                        <img src="../../static/img/engineer/7577right.png" alt="Construction Image 2">
                                    </div>
                                    <!-- <figcaption></figcaption> -->
                                </div>

                                <h3>3. Test</h3>
                                <p>After picking the positive bacteria for small scale shake flask culture, they were incubated in phosphate gradient medium and incubated at 33Â°C for 14 days and the medium was changed on the 7th day. Bacterial fluids were collected at different time points between 3 h and 14 days, placed under a fluorescence microscope, and red fluorescent bacteria were counted under 588 nm light excitation.</p>

                                <table class="data-table">
                                    <thead>
                                        <tr>
                                            <th>Simulated environment</th>
                                            <th>Phosphate concentration</th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td>Hydrogel</td>
                                            <td>1.82Ã10<sup>-3</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>Eyeball</td>
                                            <td>10<sup>-3</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>Soil</td>
                                            <td>10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>Water</td>
                                            <td>8Ã10<sup>-7</sup> mol/L</td>
                                        </tr>
                                    </tbody>
                                </table>

                                <h3>4. Learn</h3>
                                <p>The suicide switch is effective if the bacteria are able to survive (emit red fluorescence) in a considerable concentration of phosphate medium and do not survive on a low concentration of phosphate medium.</p>
                            </div>
                        </section>

                        <section id="5references" class="sub-section">
                            <div class="section-header">
                                <h2>References</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <div class="reference-list">
                                    <div class="reference-item" id="5ref1">
                                        <span class="ref-number">[1]</span>
                                        <span class="ref-content">Deng H, Zhang X, Su S, et al. Poly(2-hydroxyethyl methacrylate-co-methacrylated hyaluronan-Î²-cyclodextrin) hydrogel: A potential contact lens material with high hydrophilicity, good mechanical properties and sustained drug delivery.[J]. International journal of biological macromolecules, 2024, 283(Pt 1): 137579.</span>
                                        <a href="https://doi.org/10.1016/j.ijbiomac.2024.137579" class="ref-link" target="_blank">DOI: 10.1016/j.ijbiomac.2024.137579</a>
                                    </div>
                                </div>
                            </div>
                        </section>
                        
                    </div>
                </section>

                <section id="auxo" class="parent-section">
                    <div class="section-header">
                        <h1>Auxotrophic Mutual Assistance System</h1>
                        <button class="section-collapse-btn">
                            <ion-icon name="chevron-up-outline"></ion-icon>
                        </button>
                    </div>
                    <div class="section-content">
                        <section id="6cycle1" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 1: Survival of strain A and strain B in the absence of relevant amino acids</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Design</h3>
                                <p>We first tested the survival of E. coli strain A and E. coli strain B in the absence of their respective amino acids (Trp or His).</p>
                                <ul>
                                    <li>group A: Cultivate strain A of Escherichia coli in a medium lacking tryptophan (Trp) to observe whether it can grow.</li>
                                    <li>B group: Cultivate B. coli in a medium lacking histidine (His) and observe whether it can grow.</li>
                                    <li>Control group: Bacteria A and B are cultured in media containing their respective required amino acids, serving as positive controls.</li>
                                </ul>
        
                                <h3>2. Construction</h3>
                                <p>Construct pdCas9-HisAgRNA recombinant plasmid and pdCas9-trpCgRNA recombinant plasmid. pET-32a(+)-trpEDCBA-aroG recombinant plasmid and pET-32a(+)-HisGene recombinant plasmid.</p>
                                
                                <div class="img-grid-with-caption">
                                    <div class="img-grid">
                                        <img src="../../static/img/engineer/auxo1.png" alt="Construction Image 1">
                                        <img src="../../static/img/engineer/auxo2.png" alt="Construction Image 2">
                                    </div>
                                    <div class="img-grid">
                                        <img src="../../static/img/engineer/auxo3.png" alt="Construction Image 3">
                                        <img src="../../static/img/engineer/auxo4.png" alt="Construction Image 4">
                                    </div>
                                </div>
        
                                <p>The pdCas9-HisAgRNA recombinant plasmid and the pET-32a(+)-trpEDCBA-aroG recombinant plasmid were co-transformed into the competent SHuffle T7-B E. coli A strain that had already been transformed with pET-32a(+)-RNF114-crimso using heat shock, followed by streaking for revival. They were cultured on media containing ampicillin, and positive colonies were selected and transferred to liquid media for cultivation at 33Â°C.</p>
                                
                                <p>The pdCas9-trpCgRNA recombinant plasmid and the pET-32a(+)-HisGene recombinant plasmid were co-transformed into the competent SHuffle T7-B E. coli A strain that had already been transformed with pET-32a(+)-RNF114-crimso using heat shock, followed by streaking for revival. They were cultured on media containing ampicillin, and positive colonies were selected and transferred to liquid media for cultivation at 33Â°C.</p>
        
                                <h3>3. Test</h3>
                                <p>Select strain A and strain B, and culture them separately in liquid media lacking tryptophan or histidine at a temperature of 33Â°C for a period of 72 hours.</p>
        
                                <table class="data-table">
                                    <thead>
                                        <tr>
                                            <th>group</th>
                                            <th>tryptophan/histidine concentration</th>
                                        </tr>
                                    </thead>
                                    <tbody>
                                        <tr>
                                            <td>1</td>
                                            <td>2.00Ã10<sup>-4</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>2</td>
                                            <td>1.43Ã10<sup>-4</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>3</td>
                                            <td>1.03Ã10<sup>-4</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>4</td>
                                            <td>7.40Ã10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>5</td>
                                            <td>5.30Ã10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>6</td>
                                            <td>3.80Ã10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>7</td>
                                            <td>2.70Ã10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>8</td>
                                            <td>1.90Ã10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>9</td>
                                            <td>1.40Ã10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>10</td>
                                            <td>1.00Ã10<sup>-5</sup> mol/L</td>
                                        </tr>
                                        <tr>
                                            <td>eyeball environment</td>
                                            <td>2.00Ã10<sup>-7</sup> mol/L</td>
                                        </tr>
                                    </tbody>
                                </table>
        
                                <p>Samples were taken at different time points (0 hours, 24 hours, 48 hours, 72 hours) and colony counts or optical density (OD) measurements were conducted to observe whether bacteria could grow.</p>
        
                                <h3>4. Learn</h3>
                                <p>Strain A cannot grow in a medium without Trp, and strain B cannot grow in a medium without His, proving that they cannot survive independently and must rely on the amino acids provided by each other. The control groups of strains A and B can grow normally under conditions where the corresponding amino acids are supplemented. This indicates that the nutritional deficiency type E. coli AB has been successfully constructed.</p>
                            </div>
                        </section>
        
                        <section id="6cycle2" class="sub-section">
                            <div class="section-header">
                                <h2>Cycle 2: Co-cultivation of strain A and strain B (at different dilution concentrations)</h2>
                                <button class="section-collapse-btn">
                                    <ion-icon name="chevron-up-outline"></ion-icon>
                                </button>
                            </div>
                            <div class="section-content">
                                <h3>1. Goal</h3>
                                <p>The test examines whether bacteria A and B can form a mutualistic system through metabolic complementation at different cell densities (dilution concentrations), or if they cannot survive due to excessive distance between the bacteria.</p>
        
                                <h3>2. Design</h3>
                                <p>In the co-culture system, strain A provides histidine (His), and strain B provides tryptophan (Trp).</p>
                                <ol>
                                    <li>Set different bacterial dilution concentrations to adjust the average distance between bacteria:
                                        <ul>
                                            <li><p>High concentration group (original bacterial solution): Bacteria A and B are mixed and cultured directly without dilution.</p></li>
                                            <li><p>Medium concentration group (10 times dilution): Bacteria A and B are diluted 10 times respectively and then mixed for cultivation.</p></li>
                                            <li><p>Low concentration group (100 times dilution): Bacteria A and B are diluted 100 times respectively and then mixed for cultivation.</p></li>
                                            <li><p>Ultra-low concentration group (1000 times dilution): Bacteria A and B are diluted 1000 times respectively and then mixed for cultivation.</p></li>
                                        </ul>
                                    </li>
                                </ol>
        
                                <h3>3. Test</h3>
                                <ol>
                                    <li>Select strain A and strain B, and perform gradient dilution co-culture in a liquid medium containing minimal Trp and His (only to maintain initial growth).</li>
                                    <li>Cultured at 33Â°C for 72 hours, samples were taken at different time points (0 h, 24 h, 48 h, 72 h) and subjected to:
                                        <ul>
                                            <li>Colony count (CFU/ml), observed under a fluorescence microscope, counting red fluorescent bacteria under 588 nm light excitation.</li>
                                            <li>OD600 measurement (growth curve)</li>
                                            <li>Observed under a fluorescence microscope, examining the distribution of bacteria under 588 nm light excitation.</li>
                                        </ul>
                                    </li>
                                    <li>Record the growth of the microbial bodies and analyze the mutual growth conditions of strain A and strain B at different dilution concentrations.</li>
                                </ol>
        
                                <h3>4. Learn</h3>
                                <p>Cycle 2 determined the survival status of bacteria A and bacteria B at different dilution concentrations, allowing for the plotting of a curve of dilution concentration versus survival rate (failure rate of the nutrient mutualism system), thereby analyzing whether leakage in the eye or other environments could cause the nutrient mutualism system to fail.</p>
                            </div>
                        </section>
                </section>
                

                

            </div>
        </div>

        <!-- Footer -->
        <footer id="footer">
            <div class="logo_wrap">
                <img id="team_logo" src="../../static/img/Logo-round-text.svg" alt="">
            </div>
            <div class="follow">
                <div class="title">Follow Us</div>
                <ul class="follow_list">
                    <li><a href="https://space.bilibili.com/700401359?spm_id_from=333.337.search-card.all.click" target="_blank"><img src="https://static.igem.wiki/teams/4202/wiki/t-zju-china-bilibili.png" alt=""></a></li>
                    <li><a href="https://twitter.com/ZjuChina" target="_blank"><img src="https://static.igem.wiki/teams/4202/wiki/t-zju-china-twitter.png" alt=""></a></li>
                    <li><a href="https://mp.weixin.qq.com/s/yTMIiH4AQ2skqP-7cXr7Ig" target="_blank"><img src="https://static.igem.wiki/teams/4202/wiki/t-zju-china-wechat.png" alt=""></a></li>
                </ul>
            </div>
            <div class="contact">
                <div class="title">Contact Us</div>
                <div class="email">ZJU_China@outlook.com</div>
            </div>
        </footer>

        <script src="../../static/js/basic.js"></script>
        <script src="../../static/js/pages.js"></script>
        <script src="../../static/js/pdbviewer.js"></script>
        <script src="../../static/js/components.js"></script>
    </body>
</html>